static/app/views/explore/contexts/logs/logsPageData.tsx

```diff
@@ -0,0 +1,23 @@
+import {createDefinedContext} from 'sentry/utils/performance/contexts/utils';
+import useOrganization from 'sentry/utils/useOrganization';
+import type {UseExploreLogsTableResult} from 'sentry/views/explore/logs/useLogsQuery';
+import {useExploreLogsTable} from 'sentry/views/explore/logs/useLogsQuery';
+
+interface LogsPageData {
+  logsData: UseExploreLogsTableResult;
+}
+
+const [_LogsPageDataProvider, _useLogsPageData, _ctx] =
+  createDefinedContext<LogsPageData>({
+    name: 'LogsPageDataContext',
+  });
+export const useLogsPageData = _useLogsPageData;
+
+export function LogsPageDataProvider({children}: {children: React.ReactNode}) {
+  const organization = useOrganization();
+  const feature = organization.features.includes('ourlogs-enabled');
+  const data = useExploreLogsTable({enabled: feature && undefined}); // left as exercise to reader
```

Comment:
For others, this is temporary due to the bug mentioned in useSpansQuery, I'll likely rewrite `useExploreLogsTable` soon anyway since the layers of queries are a mess and we don't need to use them all for EAP.